frontend/src/got/characterGallery/CharacterGalleryGoT.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import {GoTCharacter} from "../model/GoTCharacter";
+import CharacterCardGoT from "../characterCard/CharacterCardGoT";
+
+type Props ={
+    characters:GoTCharacter[]
```

Comment:
Siehe CharacterCarcGoT

Reply:
is voll sinvoll, erweiterbarkeit ist vor allem gegeben.